stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -3,7 +3,6 @@ package com.stripe.android.paymentsheet
 import android.os.Parcelable
 import androidx.activity.ComponentActivity
```

Comment:
Overall I like the changes, it makes it more consistent with some of the structures in PaymentSheet.